project/project.ts

```diff
@@ -356,6 +356,10 @@ export const makeTsGdProject = async (
           // Chokidar is inconsistent about whether it passes in stats or not -
```

Comment:
Should this comment then be moved down to stay with the fs.statSync?

Reply:
Moved.